File: server/src/main/java/com/example/server/dto/notifications/NotificationResponseDto.java
package com.example.server.dto.notifications;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data @Builder @NoArgsConstructor @AllArgsConstructor public class NotificationResponseDto
{
  private String userId;
  private String userRole;
  private String content;
  private LocalDateTime timestamp;
  private String subjectId;
  private String type;
}

File: client/client/Dto/Notification/NotificationResponseDto.cs
namespace client.Dto.Notification;

public class NotificationResponseDto
{
    public string UserId { get; set; }
    public string UserRole { get; set; }
    public string Content { get; set; }
    public DateTime Timestamp { get; set; }
    public string SubjectId { get; set; }
    public string Type { get; set; } 
}